mkdocs.yml

```diff
@@ -114,22 +128,31 @@ plugins:
   - social
   - mknotebooks
   - tags:
-      tags_file: tutorials/index.md
+      tags_file: guides/tutorials/index.md
 
 
 nav:
-  - Argilla: index.md
+  - Home: index.md
   - Getting started:
     - Quickstart: getting_started/quickstart.md
     - Installation: getting_started/installation.md
     # - FAQ: getting_started/faq.md
-  - How-to guides:
-    - how_to_guides/index.md
-    - Manage users and credentials: how_to_guides/user.md
-    - Manage workspaces: how_to_guides/workspace.md
-    - Manage and create datasets: how_to_guides/dataset.md
-    - Add, update, and delete records: how_to_guides/record.md
-    - Query, filter, and export records: how_to_guides/query_export.md
+  - Guides:
+    - guides/index.md
+    - How-to guides:
+      - guides/how_to_guides/index.md
+      - Manage users and credentials: guides/how_to_guides/user.md
+      - Manage workspaces: guides/how_to_guides/workspace.md
+      - Manage and create datasets: guides/how_to_guides/dataset.md
+      - Add, update, and delete records: guides/how_to_guides/record.md
+      - Query, filter, and export records: guides/how_to_guides/query_export.md
+    - Tutorials:
+      - guides/tutorials/index.md
+      - Bulk labelling through vector search and fine-tuning a few-shot text classifier: guides/tutorials/few-shot-textcat-with-setfit.ipynb
+    - Integrations:
+      - guides/integrations/index.md
+      - distilabel ↗: https://github.com/argilla-io/distilabel
+      - argilla-llama-index ↗: https://github.com/argilla-io/argilla-llama-index
```

Comment:
These integrations are misleading because they use the legacy SDK.

Reply:
I will disable these pages for now, it was mostly to get a feel of where to potentially add these kind of things. I decided to just make it redirects to have less blato.

Reply:
Makes sense. I think it's ok to have noted out sections in these menus as the docs grow.